docker/build.sh

```diff
@@ -52,16 +65,20 @@ cp ../setup.py ./gQuant
 cp ../LICENSE ./gQuant
 rsync -av --progress ../notebooks ./gQuant --exclude data --exclude .cache --exclude many-small --exclude storage --exclude dask-worker-space --exclude __pycache__
 
+gquant_ver=$(grep version gQuant/setup.py | sed "s/^.*version='\([^;]*\)'.*/\1/")
+D_CONT=${D_CONT:="gquant/gquant:${gquant_ver}_${OS_STR}_${CUDA_VERSION}_${RAPIDS_VERSION}"}
```

Comment:
This version string looks a bit off. I built for Ubuntu 18 with CUDA 10.1.2 and version came out like this:
```
gquant/gquant:0.3_ubuntu18.04_3_0.11
```
The `CUDA_VERSION` variable isn't actually the cuda version, it's the option for which cuda version to use. The cuda version is indicated by the `CONTAINER_VER`. It should look like this:
```
D_CONT=${D_CONT:="gquant/gquant:${gquant_ver}_${OS_STR}_cuda${CONTAINER_VER}_rapids${RAPIDS_VERSION}"}
```
Which will produce:
```
gquant/gquant:0.3_ubuntu18.04_cuda10.1_rapids0.11
```

Reply:
fixed